app/lib/features/onboarding/pages/login_page.dart

```diff
@@ -133,16 +133,12 @@ class _LoginPageState extends ConsumerState<LoginPage> {
                           style: Theme.of(context).textTheme.labelLarge,
                           validator: (val) {
                             if (val == null || val.trim().isEmpty) {
-                              return AppLocalizations.of(context)!.emptyUsername;
+                              return AppLocalizations.of(context)!
+                                  .emptyUsername;
                             }
                             return null;
                           },
-                          onChanged: (value) {
-                            username.text = value;
-                            username.selection = TextSelection.fromPosition(
-                              TextPosition(offset: username.text.length),
-                            );
-                          },
+                          onChanged: (value) {},
```

Comment:
do we need these at all then? [isn't this optional](https://api.flutter.dev/flutter/material/TextFormField/onChanged.html) ?
```suggestion
```

Reply:
Yes,` onChanged: (value) {},` is optional and can be remove.